Source/m_menu.c

```diff
@@ -2591,12 +2591,13 @@ static int G_GotoNextLevel(void)
 #define X_BUTTON 301
 #define Y_BUTTON   3
 
-// Definitions of the (in this case) four key binding screens.
+// Definitions of the (in this case) five key binding screens.
```

Comment:
OMG, we have done it! :grin:

Reply:
BTW, I think we could reorganize the automap menu a bit like Pr +. Maybe add a transparent overlay mode option.

Reply:
Yes, okay.

Reply:
I think the automap menu and transparent mode will be in separate PR. This PR is already changing a lot.

Reply:
Okay, so we still need to rearrange the menu here.

Reply:
I made translucent overlap mode, but it uses `main_tranmap`. Maybe we should make a separate tranmap with a filter percentage option but I am not sure if it's worth it.

Reply:
Hm, could you show a screenshot with the default translucency settings?

Reply:
66%:
![doom01](https://user-images.githubusercontent.com/5077629/121675406-8f8e8f80-cadd-11eb-9ee1-8d30268164b5.png)
translucency off:
![doom00](https://user-images.githubusercontent.com/5077629/121675468-a6cd7d00-cadd-11eb-8096-12a3ee49dfb5.png)
33%:
![doom02](https://user-images.githubusercontent.com/5077629/121675493-ad5bf480-cadd-11eb-8beb-e71e46a61b38.png)

So, I guess we need a separate tranmap or scrap the idea.

Reply:
Okay, thank you! The difference is indeed only barely visible, at least in still images. I wouldn't say "scrap the idea" but maybe postpone for later. ;)

Reply:
> postpone for later

I agree this feature is very minor. To be honest, I didn't know Crispy and Pr+ had these features all these years. :smile: